lib/std/ascii.zig

```diff
@@ -271,7 +274,27 @@ test "ascii character classes" {
     testing.expect('c' == toLower('C'));
     testing.expect(isAlpha('c'));
     testing.expect(!isAlpha('5'));
-    testing.expect(isSpace(' '));
+    testing.expect(isWhiteSpace(' '));
+}
+
+pub fn trimWhiteSpace(buf: []const u8) []const u8 {
+    const whitespace = comptime blk: {
+        var hack = struct { storage: anytype }{ .storage = &[0]u8{} };
```

Comment:
It would be cool if we could have normal variables of type `anytype` at comptime to avoid this kind of hack.